src/kirby/components/floating-action-button/floating-action-button.component.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+
+import { FloatingActionButtonComponent } from './floating-action-button.component';
+
+describe('FloatingActionButtonComponent', () => {
```

Comment:
Missing unit tests for the various input, and the impact of binding those properties in the template.

Reply:
I have added the missing tests. I hope to cover all ground now. A small note: I couldn't find any meaning in testing the functionality of custom elements, such as <ion-fab-button> and more particularly the 'disabled' attribute in this case.